docs/pages/components/text.mdx

```diff
@@ -37,7 +37,7 @@ import { Text } from '@hack4impact-uiuc/bridge'
 <Text type="alert" as="p"> use this in alerts, semantically p</Text>
 <Text type="table" as="p"> use this in tables, semantically p </Text>
 <Text type="form" as="p"> use this in forms, semantically p </Text>
-<Text type="quote" as="p">use this in quotes: "sorry I'm late, I didnt come", semantically p</Text>
+<Text type="quote" as="p">use this in quotes: "sorry I'm late, I didn't want to come", semantically p</Text>
```

Comment:
🍵

Reply:
The reflecting code in the code block should probably be changed though :)

Reply:
good point haha woops